### PR TITLE
Release 0.0.2

### PR DESCRIPTION
**Pull request by danielgtaylor**
This continues https://github.com/istreamlabs/javascript-sdk/pull/1 but makes the following changes:

1. Use a pinned `@openapitools/openapi-generator-cli` so we all have the same one that is automatically installed with `npm install` and used by npm scripts.
2. Download and generate the SDK via `npm run update`
3. Saves the `openapi.json` into the repo, for repeatable builds
4. Renames the package to `@istreamplanet/isp`
5. Marks the package as `private` for now until we have a better story around publishing
6. Turns `publish.sh` into `release.md` with instructions about a multi-step release process we can later automate with an eye toward producing a packaged tarball you can `npm install` right now
7. The updated process uses a unique branch name for each release, hence the new PR. This prevents issues with stale local branches that introduces extra management we can easily avoid with unique branches.

### CODE REVIEW

File: list-channels.js [hunk 1]
```diff
@@ -1,47 +1,49 @@
-const IStreamPlanetChannelsApi = require('i_stream_planet_channels_api');
+const isp = require("@istreamplanet/isp");
```
Comment (lgarrett-isp) [inline list-channels.js:1]: Let's make sure we have @shaunflagg review this change--I believe this specific file was mentioned/shared with WM as an example, so we should make sure we aren't breaking anyone that is exploring ChannelAPI.

File: .openapi-generator/FILES [hunk 2]
```diff
@@ -1,6 +1,5 @@
 .babelrc
```
Comment (lgarrett-isp) [inline .openapi-generator/FILES:1]: Any chance you have a short description of what `FILES` represents in the overall workflow? When I was generating this locally I saw the `test/...` stuff getting removed here and I wasn't sure what that meant.

Comment (danielgtaylor) [reply .openapi-generator/FILES:1]: Seems to be poorly documented, but from this I guess it's purely informational and useful for some CI scenarios:

```
     * Generates a file at .openapi-generator/FILES to track the files created by the user's latest run.
     * This is ideal for CI and regeneration of code without stale/unused files from older generations.
     *
     * @param files The list tracking generated files
```

https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultGenerator.java#L1450-L1455

File: list-channels.js (hunk 1 above)
Comment (shaunflagg) [reply list-channels.js:1]: LGTM. Thanks for the update. That 'i_stream_planet_channels_api' was ugly.

Comment (shaunflagg) [reply list-channels.js:1]: And I can update list_channels.js to align with this change...

**Pull request merged**